clients/cli/devices.c

```diff
@@ -2640,24 +2815,20 @@ do_device_wifi_list (NmCli *nmc, int argc, char **argv)
 {
 	GError *error = NULL;
 	NMDevice *device = NULL;
-	NMAccessPoint *ap = NULL;
 	const char *ifname = NULL;
 	const char *bssid_user = NULL;
+	const char *rescan = NULL;
 	gs_free NMDevice **devices = NULL;
-	const GPtrArray *aps;
-	APInfo *info;
-	int i, j;
+	int i; //, j;
```

Comment:
^^^

Reply:
and lets not use signed integers for loop variables that shall not be negative?

Reply:
Ok.